cypress/e2e/shared/buckets.test.ts

```diff
@@ -132,7 +132,12 @@ describe('Buckets', () => {
   })
 
   describe('add data', function() {
+    const TELEGRAF_SYSTEMS_PLUGINS_ORIGINAL_COUNT = 5
+
     it('configure telegraf agent', () => {
+      cy.setFeatureFlags({
+        telegrafUiRefresh: true,
+      })
```

Comment:
For now, OSS needs this flag set because the flag is still in use in the code. This piece of code will be removed as part of removing the feature flag.